xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/XWikiLivedata.vue

```diff
@@ -25,7 +25,7 @@
   totally autonomous.
 -->
 <template>
-  <div class="xwiki-livedata">
+  <div class="xwiki-livedata" :id="data.id">
```

Comment:
I'm curious: is this related to this PR or is it some limitation/bug that we had on the LD component?

Reply:
Adding the id is needed to select the right DOM element in the page.